interpose.c

```diff
@@ -0,0 +1,49 @@
+#include "interpose.h"
```

Comment:
Did you test this to see if it prints `"[f]open called ...`"?

Reply:
yes, tested.